dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -20,7 +20,7 @@ class ReportAbuseController < ApplicationController
   AGE_CUSTOM_FIELD_ID = 24_024_923
 
   def report_abuse
-    unless Rails.env.development?
+    unless Rails.env.development? || Rails.env.test?
```

Comment:
<img width="1162" alt="screen shot 2019-01-08 at 1 04 18 pm" src="https://user-images.githubusercontent.com/12300669/50858970-e27d1f00-1346-11e9-9a9c-2ede8f570517.png">
We're blocked from sending Zendesk tickets, which reporting abuse does, from the test environment; so, I check the environment before submitting the ticket from the form.